receiver/sqlserverreceiver/scraper.go

```diff
@@ -194,3 +196,41 @@ func (s *sqlServerScraperHelper) recordDatabasePerfCounterMetrics(ctx context.Co
 
 	return errors.Join(errs...)
 }
+
+func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context, rb *metadata.ResourceBuilder) error {
+	// Constants are the column names of the database status
+	const dbOnline = "db_online"
+	const dbRestoring = "db_restoring"
+	const dbRecovering = "db_recovering"
+	const dbPendingRecovery = "db_recoveryPending"
+	const dbSuspect = "db_suspect"
+	const dbOffline = "db_offline"
+
+	rows, err := s.client.QueryRows(ctx)
+
+	if err != nil {
+		if errors.Is(err, sqlquery.ErrNullValueWarning) {
+			s.logger.Warn("problems encountered getting metric rows", zap.Error(err))
+		} else {
+			return fmt.Errorf("sqlServerScraperHelper: %w", err)
```

Comment:
small nit: may be add more info into error is clearer?
for example, fmt.Errorf("sqlServerScraperHelper failed getting metric rows: %w", err)